akeyless/resource_role_test.go

```diff
@@ -63,6 +63,8 @@ func TestRoleWithAssocResourceUpdate(t *testing.T) {
 				path = "/terraform-tests/*"
 				rule_type = "auth-method-rule"
 			}
+			audit_access = "all"
+  			analytics_access = "all"
```

Comment:
Maybe test "own" as well
```suggestion
  			analytics_access = "own"
```

Same for others